roles/container-engine/containerd/tasks/main.yml

```diff
@@ -0,0 +1,50 @@
+---
+- name: Fail containerd setup if distribution is not supported
+  fail:
+    msg: "{{ ansible_distribution }} is not supported by containerd."
+  when:
+    - not ansible_distribution in ["CentOS","RedHat", "Ubuntu", "Debian"]
+
+- name: Install Docker
```

Comment:
Am I right that you're installing Docker just to bring ContainerD as a dependency? Why? Containerd exists as a separate package, also it doesn't allow to override package properly (with a custom version, for example)

Reply:
Thanks,

Independent containerd package repo and name is different for each distribution.
Briefly, I just include docker for initial contribution. But it could be improved after merge.